cognito-pre-token-generation/src/test/java/no/unit/nva/cognito/service/CustomerDbClientTest.java

```diff
@@ -0,0 +1,70 @@
+package no.unit.nva.cognito.service;
+
+import no.unit.nva.cognito.model.CustomerResponse;
+import no.unit.nva.customer.model.CustomerDb;
+import no.unit.nva.customer.model.CustomerMapper;
+import no.unit.nva.customer.service.impl.DynamoDBCustomerService;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.apigateway.exceptions.NotFoundException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+import java.util.UUID;
+
+import static java.lang.String.join;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class CustomerDbClientTest {
```

Comment:
I suppose the tests are OK for now, but should we create a task to replace these tests with a "real" service talking to a local DynamoDB?

Reply:
Yes, we can create separate task for this.